english/make-friends/where_am_i/where_am_i.md

```diff
@@ -0,0 +1,18 @@
+You realise you need as much information as you need to get out of this situation. And the only thing you can ask about this seems to be Marshmellow man. You decide to befriend him.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.